### DISCUSSION TITLE
Idea: Realistic challenges with colonizing planets.

### DISCUSSION THREAD

**Discussion by Asteroid4** [TeamGalacticraft/Galacticraft#228] [Feature Requests]
In galacticraft 4, you can go to mars with only a simple breathing mask and be fine. My idea is to add space suits that can be optimized for different atmospheres, like extra planets. Also, planets have unique challenges to them in real life, and I think adding some of these will make a more interesting experience.

**Comment by vos6434** [comment]
Adding an armour defeats the purpose of the GC inventory. The GC inventory exists so players can wear any armour the feel like using.

**Reply by Asteroid4** [reply]
Space suits would be equipped by putting them in the GC inventory, like the space suits in extra planets.

**Comment by Asteroid4** [comment]
To clarify, by "unique challenges", I mean that, for example, mars is covered in dust. It's really easy to track dust into every corner of your mars base, and the dust is toxic. There are no other planets like that, I think. You would need some way to deal with that in galacticraft.
